### PR TITLE
Upstream v6 changes doc and config samples

### PR DESCRIPTION
**Pull request by cahartma**
### Description
Updated docs snippets and made all examples into complete specs.   I still need to figure out what Libby wants to do with the samples.adoc 

/cc @Clee2691 @vparfonov
/assign @jcantrill

### CODE REVIEW

File: docs/administration/upgrade/v6.0_changes.adoc [hunk 1]
```diff
@@ -566,78 +480,132 @@ spec:
...
+      inputRefs:
         - application
-      name: to-default
-      parse: json
+        - infrastructure
+      filterRefs:
+        - my-parse
       outputRefs:
-        - default
+        - es-managed
```
Comment (jcantrill) [inline docs/administration/upgrade/v6.0_changes.adoc]: this does not match the output name

File: docs/administration/upgrade/v6.0_changes.adoc [hunk 2]
```diff
@@ -566,97 +480,162 @@ spec:
...
-----
-.v6 ClusterLogForwarder
+  - name: my-pipeline
+    inputRefs:
+    - application
+    - infrastructure
+    filterRefs:
+    - my-parse
+    outputRefs:
+    - es-managed
```
Comment (jcantrill) [inline docs/administration/upgrade/v6.0_changes.adoc:562]: name ref is incorrect

File: docs/administration/upgrade/v6.0_changes.adoc [hunk 3]
```diff
@@ -566,97 +480,162 @@ spec:
...
-        ca:
-          key: ca-bundle.crt
-          secretName: collector
-        certificate:
-          key: tls.crt
-          secretName: collector
-        key:
-          key: tls.key
-          secretName: collector
+  - name: es-managed-app
```
Comment (jcantrill) [inline docs/administration/upgrade/v6.0_changes.adoc:587]: I just thought about this, we could be clever and allow a single output def that relies upon an openshift.label to craft the index.  We create 2 pipelines to the same output and use the openshiftLabel filter on each pipeline to set "app" or "infra" and then use that for the index name

File: docs/reference/samples/observability.filters-pipelines-unchanged.yaml [hunk 4]
```diff
@@ -28,4 +28,4 @@ spec:
     - name: my-output
```
Comment (jcantrill) [inline docs/reference/samples/observability.filters-pipelines-unchanged.yaml:28]: We should consider naming any yaml which is not complete in such a way to indicate its not complete

Comment (cahartma) [reply docs/reference/samples/observability.filters-pipelines-unchanged.yaml:28]: Include "Snippet" in the title, or what I prefer is the ellipses top and bottom?

File: docs/administration/upgrade/v6.0_changes.adoc (hunk 3 above)
Comment (alanconway) [reply docs/administration/upgrade/v6.0_changes.adoc:587]: That is clever :)

Comment (cahartma) [reply docs/administration/upgrade/v6.0_changes.adoc:587]: Perfect, and I was also thinking, if they have created custom indexes using the 'structuredType' options in 5.9, we may want to show how to achieve that as well.

Comment (cahartma) [reply docs/administration/upgrade/v6.0_changes.adoc]: Updated, please review.

File: docs/reference/samples/observability.filters-pipelines-unchanged.yaml (hunk 4 above)
Comment (cahartma) [reply docs/reference/samples/observability.filters-pipelines-unchanged.yaml:28]: Or do you mean the actual filename?   These should all be complete at this point.

File: docs/reference/samples/observability.inputs-receivers.yaml [hunk 5]
```diff
@@ -44,3 +44,24 @@ spec:
...
+  outputs:
+    - name: my-http-output
+      type: http
+      http:
+        url: http://my-log-output:80
+    - name: my-sys-out
+      type: syslog
+      syslog:
+        rfc: RFC5424
+        url: tcp://127.0.0.1:24224
```
Comment (anpingli) [inline docs/reference/samples/observability.inputs-receivers.yaml:67]: how about  tcp://my-syslogservice.example.com:514

**Review by jcantrill — COMMENTED** [review]
/lgtm

File: docs/reference/samples/observability.inputs-receivers.yaml (hunk 5 above)
Comment (cahartma) [reply docs/reference/samples/observability.inputs-receivers.yaml:67]: Will update for future if you think it's helpful.  As long as the current example passes validation, we are good for now.

**Pull request merged**